spyder/plugins/appearance/widgets.py

```diff
@@ -8,7 +8,9 @@
 
 from qtpy.QtCore import Qt
```

Comment:
```suggestion
import re

from qtpy.QtCore import Qt
```

`re` is a standard library module, so it needs to be placed before third-party ones.